concordium-contracts-common/src/types.rs

```diff
@@ -392,6 +393,43 @@ impl AccountBalance {
     pub fn available(&self) -> Amount { self.total - cmp::max(self.locked, self.staked) }
 }
 
+#[derive(Debug)]
+pub enum AccountKind {}
+
+/// The minimum number of credentials that need to sign any transaction coming
+/// from an associated account.
+pub type AccountThreshold = Threshold<AccountKind>;
+
+#[derive(Debug)]
+pub enum SignatureKind {}
+
+/// The minimum number of signatures on a credential that need to sign any
+/// transaction coming from an associated account.
+pub type SignatureThreshold = Threshold<SignatureKind>;
```

Comment:
If I understand correctly, then `AccountThreshold` comes from `base` (old definition) while the `SignatureThreshold` from `concordium-std`. Both types behave technically the same but we cannot unify them because different `signature types` are already defined in different places.

Maybe we can add a note here that these two things are the same.

Reply:
No, both come from concordium-base. I've added a comment about it though.